Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddKeyAsNewOwnerViewController.swift

```diff
@@ -24,12 +24,19 @@ class AddKeyAsNewOwnerViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        presentationController?.delegate = self
+
         titleLabel.setStyle(.primary)
         descriptionLabel.setStyle(.secondary)
         addButton.setText("Add as owner", .filled)
         skipButton.setText("Skip", .plain)
     }
 
+    // Called when user swipes down the modal screen
+    func presentationControllerWillDismiss(_ presentationController: UIPresentationController) {
```

Comment:
will dismiss or did dismiss? Does it trigger if the swipe not done fully?

Reply:
good catch. We had this type issue recently